--- node_modules/diagrams/src/vendors/flowchart.js.old	2019-08-13 10:28:31.083834768 +0200
+++ node_modules/diagrams/src/vendors/flowchart.js	2019-08-13 10:26:53.435582179 +0200
@@ -101,7 +101,7 @@
     'line-color': 'black',
     'element-color': 'black',
     'fill': 'white',
-    'yes-text': 'yes',
+    'yes-text': 'sí',
     'no-text': 'no',
     'arrow-end': 'block',
     'class': 'flowchart',
